lib/get-format.js

```diff
@@ -71,6 +80,9 @@ function getFileProtocolModuleFormat(url) {
   const format = extensionFormatMap[ext]
   if (format) return format
 
+  // Taken from https://github.com/nodejs/node/blob/master/lib/internal/modules/esm/get_format.js#L66-L67
```

Comment:
I can remove the line comments since they might be too noisy.

Reply:
And it's already present at the beginning of the file 😉